packages/mdast/mail-templates/src/shared/components/Datawrapper.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { EdgeToEdgeFigure, Figure } from './Figure'
+
+// We generate the following images in Datawrapper upon publishing:
+//  1. https://datawrapper.dwcdn.net/DW123ID/mail.png -> 450px wide
+//  2. https://datawrapper.dwcdn.net/DW123ID/mailplain.png -> 450px wide, no header or footer
+//  3. https://datawrapper.dwcdn.net/DW123ID/edgetoedge.png -> 1280px wide
+//  4. https://datawrapper.dwcdn.net/DW123ID/edgetoedgeplain.png -> 1280px wide, no header or footer
+
+const Chart = ({ src, alt, width }) => (
+  <img
+    key='image'
+    style={{
+      border: '0px',
+      height: 'auto',
+      margin: '0px',
+      maxWidth: '100%',
+      width,
+    }}
+    width={width}
+    src={src}
+    alt={alt}
+  />
+)
+
+export const Datawrapper = ({ datawrapperId, alt, plain }) => (
+  <Figure>
+    <Chart
+      src={`https://datawrapper.dwcdn.net/${datawrapperId}/mail${plain ? 'plain' : ''}.png`}
+      width={450}
```

Comment:
In the [figureRule](https://github.com/republik/plattform/pull/961/files#diff-ba4293d866802fc6f9ee5581a78e8aa12b602cd8eead03a61f08ca60b1236d69R32) it's defined as 600px?

Reply:
yes, exactly. i tried sending a test email with width 600px and the labels were so tiny they were hurting my eyes. so i wanted to a width of 450px. Not sure… second opinion appreciated on this.

Reply:
OK, yes, makes sense. I was just checking because I saw two different numbers. So the 450px image will not be scaled up to 600px in the email (I have no idea what the image size in the figureRule does)?

Reply:
no – then the max width for the chart-image is 450px. as i said, i am a bit on the fence about which size a regular email chart should be...